test/recipes/80-test_cms.t

```diff
@@ -807,8 +807,8 @@ subtest "CMS binary input tests\n" => sub {
     my $input = srctop_file("test", "smcont.bin");
     my $signed = "smcont.signed";
     my $verified = "smcont.verified";
-    my $cert = srctop_file("test", "certs", "p384-root.pem");
-    my $key = srctop_file("test", "certs", "p384-root-key.pem");
+    my $cert = srctop_file("test", "certs", "ee-self-signed.pem");
```

Comment:
What caused this to change?

Reply:
Part of the CI is a configuration where ECC is not enabled, which made the tests fail.
So I changed they key and cert to something of type RSA, which is supported by all configurations.

Reply:
Ahh ok this was only for your new test - all good.